test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -381,10 +381,9 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 				);
 				await this.closeSidebar();
 			} else {
-				await driverHelper.selectElementByText(
+				await driverHelper.clickWhenClickable(
 					this.driver,
-					By.css( '.page-template-modal__buttons .components-button' ),
-					'Use Blank template'
+					By.css( '.page-template-modal__buttons .components-button' )
```

Comment:
The naming suggests there may be more buttons. This will pick the first? Should we be clear about picking the primary?

```suggestion
					By.css( '.page-template-modal__buttons .components-button.is-primary' )
```

This seems to be from:

https://github.com/Automattic/wp-calypso/blob/e86ef811a5b8473d0bbfd066b9d4d728b62a25ee/apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js#L310-L326

Introduced in #37624

cc: @Automattic/ajax @Automattic/cylon